### PR TITLE
[ca-ES] Fix some typos

### PR DESCRIPTION
**Pull request by VRoxa**

### CODE REVIEW

**Comment by abnerlee** [comment]
Sorry, but there are some conflicts now..

**Comment by VRoxa** [comment]
Solved, @abnerlee

File: ca-ES.lproj/Menu.strings [hunk 1]
```diff
@@ -238,7 +238,11 @@
 "Copy Image to" = "Copia la imatge a";
 "Open Image Location" = "Obre la ubicació de la imatge";
 
+<<<<<<< HEAD
 "Learn More" = "Aprèn més";
+=======
+"Learn More" = "Més informació";
+>>>>>>> upstream/master
```
Comment (abnerlee) [inline ca-ES.lproj/Menu.strings:245]: Would you either use "Aprèn més" or use "Més informació" ?

**Review by abnerlee — CHANGES_REQUESTED** [review]

File: ca-ES.lproj/Menu.strings (hunk 1 above)
Comment (VRoxa) [reply ca-ES.lproj/Menu.strings]: I just forgot to save this conflict.
This change was made in the previous PR. "Aprèn més" is the literal translation of "Learn more".
"Més informació" means "More information"; which is kind of more natural.

**Pull request merged**